### PR TITLE
ntpd: add page

### PR DESCRIPTION
**Pull request by gwennlbh**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 4.2.8

### CODE REVIEW

File: pages/linux/ntpd.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ntpd
+
+> The official Network Time Protocol daemon program.
```
Comment (vitorhcl) [inline pages/linux/ntpd.md:3]:
```suggestion
> The official NTP (Network Time Protocol) daemon to synchronize the system clock to remote time servers or local reference clocks.
```
We use more explanatory descriptions for an easier comprehension by begginers.

File: pages/linux/ntpd.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://www.ntp.org/documentation/4.2.8-series/ntpd/>.
+
```
Comment (vitorhcl) [inline pages/linux/ntpd.md:5]:
```suggestion

- Start the daemon:

`sudo ntpd`
```
We prefer to start with a basic usage and progressively increase the complexity.

File: pages/linux/ntpd.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+- Synchronize system time with remote servers, allowing "Big" adjustments with --panicgate:
+
+`sudo ntpd --panicgate --quit`
```
Comment (vitorhcl) [inline pages/linux/ntpd.md:8]:
```suggestion
- Synchronize system time with remote servers a single time (quit after synchronizing):

`sudo ntpd --quit`

- Synchronize a single time allowing "Big" adjustments:

`sudo ntpd --panicgate --quit`
```
Distributing the options allows an easier comprehension and more concise descriptions, that's what most tldr pages do.

**Review by vitorhcl — COMMENTED** [review]
Thank you for your contribution! This command was missing indeed and brought a good and simple example usage :)

I have some suggestions about the guidelines we generally follow. See details in https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md.

**Comment by vitorhcl** [comment]
I think https://manned.org/ntpd is more adequate for the "More information" link, because the linux version has some differences. For example, you used long options that are not on https://www.ntp.org/documentation/4.2.8-series/ntpd/.

**Comment by ewen-lbh** [comment]
> I think https://manned.org/ntpd is more adequate for the "More information" link, because the linux version has some differences. For example, you used long options that are not on https://www.ntp.org/documentation/4.2.8-series/ntpd/.

Yeah, I hesitated on that but wanted to get to the "root source" of information, but I guess that if the tldr page has linux-specific options, the manned page is better

**Comment by vitorhcl** [comment]
> Yeah, I hesitated on that but wanted to get to the "root source" of information, but I guess that if the tldr page has linux-specific options, the manned page is better

The previous URL pointed to the specification, while the current URL points to the implementation documentation, so it is more accurate about the command usage.

**Review by vitorhcl — APPROVED** [review]
LGTM!

Thank you for your your quick fixes and, again, for your contribution :)

Welcome to tldr! Feel free to contribute to the project in any way you want.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**